backend/src/v1/code-table/code-table.service.ts

```diff
@@ -493,16 +493,18 @@ export class CodeTableService {
       }
       case "equipment": {
         const { data } = await get(token, {
-          query: "{equipmentCodes{equipmentCode shortDescription longDescription displayOrder activeIndicator}}",
+          query:
+            "{equipmentCodes{equipmentCode shortDescription longDescription displayOrder activeIndicator isTrapIndicator}}",
         });
         const results = data.equipmentCodes.map(
-          ({ equipmentCode, shortDescription, longDescription, displayOrder, activeIndicator }) => {
+          ({ equipmentCode, shortDescription, longDescription, displayOrder, activeIndicator, isTrapIndicator }) => {
             const table: Equipment = {
```

Comment:
When creating a new object like you are with the table, you can simplify things a bit as you're repeating yourself.

you can rewrite this: 
```
const table: Equipment = { 
  equipment: equipmentCode,
  shortDescription: shortDescription,
  longDescription: longDescription,
  displayOrder: displayOrder,
  isActive: activeIndicator,
  isTrapIndicator: isTrapIndicator,
};
```
as this: 
```
const table: Equipment = { 
  equipment: equipmentCode,
  shortDescription,
  longDescription,
  displayOrder,
  isActive: activeIndicator,
  isTrapIndicator,
};
```

When you're creating a new object, if the property name and the variable name are the same you don't need to define the property.